docs/database-views/input4MIPs_delivery-summary_CMIP6Plus.html

```diff
@@ -60,7 +60,7 @@ <h4>
       <td><a href='http://www.globalfiredata.org' target='_blank'>Open biomass burning emissions</a></td>
       <td>DRES-CMIP-BB4CMIP7-1-0</td>
       <td>Data in preparation and final metadata checks</td>
```

Comment:
@znichollscr need to figure out where to catch this text and update it to "Preliminary dataset available"